backend/core/test/applications/applications.e2e-spec.ts

```diff
@@ -158,6 +159,11 @@ describe("Applications", () => {
         ListingsModule,
         ApplicationsModule,
         TypeOrmModule.forFeature([Application, HouseholdMember]),
+        ThrottlerModule.forRoot({
+          ttl: 60,
+          limit: 5,
+          ignoreUserAgents: [/^node-superagent.*$/],
```

Comment:
Why ignore node-superagent?

Reply:
This agent is used by all tests. Without this all tests hits rate limit and fails. Note that the test for rate limits i added in this PR is forcing different agent.